src/routes/register/components/userInfoForm.svelte

```diff
@@ -56,15 +156,33 @@
         {/if}
     </button>
 </div>
-<div class="input-group input-group-divider relative grid-cols-[auto_1fr_auto]">
+{#if passwordIndicator === indicatorStatus.warning}
+    <div class="flex flex-col items-center justify-center transition-opacity">
+        <ul class="text-sm text-warning-400">
+            <li>Das Passwort muss min. 12 Zeichen lang sein</li>
+            <li>Das Passwort muss enthalten:</li>
+            <li>- min. 1 Kleinbuchstabe</li>
+            <li>- min. 1 Großbuchstabe</li>
```

Comment:
```suggestion
            <li>- min. 1 Kleinbuchstabe (a-z)</li>
            <li>- min. 1 Großbuchstabe (A-Z)</li>
```
"Ä" z. B. ist auch ein Großbuchstabe, aber der würde hier nicht gematched werden, richtig? Deswegen die Hinweise.